### ISSUE TITLE
Name conflict

### ISSUE THREAD

**Issue by welshjf** [commx/python-rrdtool#1]
Is there any chance of getting this merged upstream (https://github.com/oetiker/rrdtool-1.x/tree/master/bindings/python)? It's a little frustrating that the two are incompatible but share the same name. For example, "rrdtool" on PyPI is this, but the rrdtool-python package on Fedora (provides python-rrdtool) is the other.

**Comment by macfreek** [comment]
I second the OP request.

A brief search found these projects:

Name | Pip package | Version | Year | Project URL | Repository URL | Author
---- | ----------- | ------- | ---- | ----------- | -------------- | -----------
py-rrdtool |  | 0.2.1 | 2003 | http://www.nongnu.org/py-rrdtool/ | [_rrdtoolmodule.c](http://cvs.savannah.gnu.org/viewvc/py-rrdtool/py-rrdtool/src/_rrdtoolmodule.c?view=annotate) | Hye-Shik Chang
py-rrdtool | py-rrdtool | 1.0b1 | 2006 | http://py-rrdtool.sourceforge.net/ | [_rrdtoolmodule.c](http://py-rrdtool.cvs.sourceforge.net/viewvc/py-rrdtool/py-rrdtool/src/_rrdtoolmodule.c?view=annotate) | Hye-Shik Chang, Mihai Ibanescu
 | python-rrdtool | 1.4.7 | 2012 | https://github.com/pbanaszkiewicz/python-rrdtool | [rrd_tool.c](https://github.com/pbanaszkiewicz/python-rrdtool/blob/master/rrdtool-1.4.7/src/rrd_tool.c) | Piotr Banaszkiewicz
rrdtool |  | 1.4.8 | 2015 | http://oss.oetiker.ch/rrdtool/prog/rrdpython.en.html | [rrdtoolmodule.c](https://github.com/oetiker/rrdtool-1.x/blob/master/bindings/python/rrdtoolmodule.c) | Hye-Shik Chang, Alan Milligan
rrdtool | rrdtool | 0.1.2 | 2015 | https://github.com/commx/python-rrdtool | [rrdtoolmodule.c](https://github.com/commx/python-rrdtool/blob/master/rrdtoolmodule.c) | Hye-Shik Chang, Christian Jurk

Both this project and the official rrdtool Python bindings seem to originate from the same source by Hye-Shik Chang. However, there seem to be some differences in syntax between the two packages, which lead to unexpected errors. It would be great if a common API could be established, one way or the other.

To add to confusion, Debian's [python-rrdtool](https://packages.debian.org/jessie/python-rrdtool) package installs the official bindings from RRDtool 1.4.8, though it's egg file still only contains reference to version 0.2.2. of Hye-Shik Chang's original implementation. PyPi's [python-rrdtool](https://pypi.python.org/pypi/python-rrdtool/) seems to refer to an older and different codebase altogether.

**Comment by commx** [comment]
**python-rrdtool** is has been merged into upstream, replacing Hye-Shik Chang's version. As mentioned in the referenced issue, I have no influence in the package naming on PyPI. Historically, I chose "rrdtool" as the package name, because "python-rrdtool" was already taken.

This project keeps being maintained to provide a pythonic way to install the bindings via pip and similar (under PyPI name "rrdtool").
